src/gateways.json

```diff
@@ -0,0 +1,8 @@
+[
+  "https://ipfs.io",
+  "https://dweb.link",
+  "https://gateway.ipfs.io",
+  "https://ninetailed.ninja",
+  "https://via0.com",
+  "https://ipfs.eternum.io"
+]
```

Comment:
this file was recently changed to .ts to prevent rebase hell that comes from a lot of in-flight PRs. I'd much rather use .ts or .js, and have trailing commas, or YAML/TOML or other format that would reduce rebase pains.

Reply:
(reworked a second time to load a txt file)